SCMS.Services.Api/Brokers/Storages/StorageBroker.StudentSchool.cs

```diff
@@ -24,5 +25,8 @@ public async ValueTask<StudentSchool> InsertStudentSchoolAsync(StudentSchool stu
 
             return entityEntry.Entity;
         }
+
+        public IQueryable<StudentSchool> SelectAllStudentSchools() 
+            => this.StudentSchools.AsQueryable();
```

Comment:
Move the fat-arrow up by one and just return the db set - you don't need the `AsQueryable`